cg/services/post_processing/error_handlers.py

```diff
@@ -0,0 +1,27 @@
+"""Module that holds the error handlers for the Post Processing service."""
+
+import logging
+from cg.exc import CgError
+
+LOG = logging.getLogger(__name__)
+
+
+def handle_post_processing_errors(to_except: tuple, to_raise):
```

Comment:
```suggestion
def handle_post_processing_errors(to_except: tuple, to_raise: Type[Exception]):
```

Reply:
It is not an Type Exception, I tried those type hints but it was complaining. I will look up waht those should be